### DISCUSSION TITLE
Simplier Table?

### DISCUSSION THREAD

**Discussion by ilcaa72** [datapane/datapane#42] [Help]
hello wondering if i can just visually plot the basic DataFrame that is visible in Jupyter.  the table that comes with DataPane is to much when you just want to display values in a table ( i dont want/need the filtering, data labels, icons, etc, etc)

is there a way to produce a basic table with bold column headers within datapane?

**Comment by lanthias** [comment]
This is a timely suggestion! We have support for the basic pandas HTML table coming in the next few days. This will allow you to have simpler tables, but also style your table in Python using DataFrame styles.

If you would like a table prior to that, you can also generate markdown tables like this, and pass them in as a string.

```python
dp.Report(
  df.to_markdown()
)
```

Thank you for the suggestion!

**Reply by ilcaa72** [reply]
thats great news, ill look for the update after thanksgiving, in the meantime the df.to_markdown() works for my needs

**Reply by lanthias** [reply]
Hi there, we've added a new simpler Table component based on feedback - you can read about it here: https://blog.datapane.com/announcing-datapane-0-9/

Let me know if I can help at all!
